src/ptvsd/adapter/__main__.py

```diff
@@ -76,10 +72,6 @@ def _parse_argv(argv):
         help=argparse.SUPPRESS
     )
 
-    parser.add_argument(
-        "--cls", action="store_true", help="clear screen before starting the debuggee"
```

Comment:
See this: https://www.reddit.com/r/vscode/comments/d6oggo/how_do_i_clear_the_terminal_screen_upon_executing/ 

We may have a case for `--cls`. But handle it via launcher? or this could be a debug config.

Reply:
Yeah, for user scenarios like that, this is definitely something that the launcher should do, triggered by a boolean flag in debug config. In the adapter, it was solely for testing the adapter itself, and even then I ended up not using it much.